include/pybind11/pybind11.h

```diff
@@ -55,8 +55,8 @@ PYBIND11_NAMESPACE_BEGIN(PYBIND11_NAMESPACE)
 /// Wraps an arbitrary C++ function/method/lambda function/.. into a callable Python object
 class cpp_function : public function {
 public:
-    cpp_function() { }
-    cpp_function(std::nullptr_t) { }
+    cpp_function();
+    cpp_function(std::nullptr_t);
```

Comment:
Same here. This might just stay as it is. The real gain comes from things like `initialize_generic`, right?